### DISCUSSION TITLE
Allow DMs without needing to join a team first

### DISCUSSION THREAD

**Discussion by chetanyakan** [mattermost/community#76] [Practical Ideas]
Needs product change.
Can be a system console setting to allow DMs and Group chats without joining a team.
In the select teams page, there could be a UI to send DMs even if there is no team available to join.

**Comment by esethna** [comment]
We actually have that config setting already to allow DMs outside your current team

**Reply by esethna** [reply]
Oh, I misunderstood, you mean be able to use mattermost for DMing only rather than needing to join a team to access DMs. We've definitely been thinking about the concept of separating DMs out so they aren't so tied to the messaging platform
